### ISSUE TITLE
[doc] 9.2.2.2  The backup location on S3-compatible storage providers.

### ISSUE THREAD

**Issue by tgeek77** [SUSE/doc-caasp#788]
9.2.2.2  The backup location on S3-compatible storage providers.

https://documentation.suse.com/suse-caasp/4.1/single-html/caasp-admin/#_the_backup_location_on_s3_compatible_storage_providers_2

Correct helm command:

```
helm install \
    --name velero \
    --namespace <NAMESPACE> \
    --set-file credentials.secretContents.cloud=credentials-velero \
    --set configuration.provider=aws \
    --set configuration.backupStorageLocation.name=default \
    --set configuration.backupStorageLocation.name=<BUCKET_NAME> \
    --set configuration.backupStorageLocation.bucket=<BUCKET_NAME> \
    --set configuration.backupStorageLocation.config.region=<REGION_NAME> \
    --set configuration.backupStorageLocation.config.s3ForcePathStyle=true \
    --set configuration.backupStorageLocation.config.s3Url=<S3_COMPATIBLE_STORAGE_SERVER_URL> \
    --set snapshotsEnabled=true \
    --set deployRestic=true \
    --set configuration.volumeSnapshotLocation.name=default \
    --set configuration.volumeSnapshotLocation.config.region=minio \
    --set initContainers[0].name=velero-plugin-for-aws \
    --set initContainers[0].image=registry.suse.com/caasp/v4/velero-plugin-for-aws:1.0.1 \
    --set initContainers[0].volumeMounts[0].mountPath=/target \
    --set initContainers[0].volumeMounts[0].name=plugins \
    suse/velero
```
The line: --set configuration.backupStorageLocation.bucket=<BUCKET_NAME> was included per an email from JenTing Hsiao in the caasp-internal mailing list on 30.4.20. Without this line, the command will fail.

**Comment by r0ckarong** [comment]
@innobead

**Comment by jenting** [comment]
Thanks for the reporting, already merged #786.
Wait for @r0ckarong to release it :bow:
